apps/target_apps_store.py

```diff
@@ -70,16 +98,28 @@ def images_size(self, target: FactoryClient.Target):
 
         return app_image_size
 
+    def _copy_encrypted(self, app_img: str, dst_images_dir: str, encrypt_apps: str):
```

Comment:
`encrypt_apps` a bit confusing param name, it's an encryption key, isn't it?

Reply:
https://github.com/foundriesio/ci-scripts/pull/131/commits/15580749455d264e0e1b0431e14cbb2001df69f3